### PR TITLE
Handle new mediacheck-select-device needle

### PR DESCRIPTION
**Pull request by michelmno**
now that linuxrc may reports a new screen with correction
https://bugzilla.suse.com/show_bug.cgi?id=1036004

need new needle as provided by https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/402

validated locally in my openQA instance with ppc64/ppc64le with TW Build 602.2

### CODE REVIEW

**Comment by okurz** [comment]
Needle merge

**Comment by mitiao** [comment]
LGTM, and verified links?

File: tests/installation/mediacheck.pm [hunk 1]
```diff
@@ -27,12 +28,18 @@ sub run {
     ensure_shim_import;
     $self->select_bootmenu_more('inst-onmediacheck', 1);
 
-    # the timeout is insane - but some old DVDs took almost forever, could
-    # recheck with all current one and lower again
-    assert_screen [qw(mediacheck-ok mediacheck-checksum-wrong)], 3600;
-    send_key "ret";
-    if (match_has_tag('mediacheck-checksum-wrong')) {
-        die "Checksum reported as wrong";
+    while ($iterations++ < 3) {
```
Comment (rwx788) [inline tests/installation/mediacheck.pm]: Is it kind of workaround? What is the reason to have a loop here?

Comment (michelmno) [reply tests/installation/mediacheck.pm]: Thank you @rwx788; this patch is not a workaround, the loop is present to continue to support the "old" linuxrc version for release other than TW that do not have yet support of this new screen.

**Pull request merged**